test/dlc_tests/container_tests/bin/pytorch_tests/setupPyTorchBackendTest

```diff
@@ -8,3 +8,4 @@ source dlbs/scripts/environment.sh
 sed -i "s/model = DDP(model, shared_param=True)/model = DDP(model, [opts['local_rank']], opts['local_rank'])/g" dlbs/python/pytorch_benchmarks/benchmarks.py
 sed -i '/from apex.fp16_utils import network_to_half, prep_param_lists, model_grads_to_master_grads, master_params_to_model_params/,/raise ImportError("Please install apex from https:\/\/www.github.com\/nvidia\/apex to run this example.")/d' dlbs/python/pytorch_benchmarks/benchmarks.py
 sed -i '/try:[ ]*$/{N;s/try:\s*from apex.parallel import DistributedDataParallel as DDP/from torch.nn.parallel import DistributedDataParallel as DDP/}' dlbs/python/pytorch_benchmarks/benchmarks.py
+sed -i 's/async=True/non_blocking=True/g' dlbs/python/pytorch_benchmarks/dataset_factory.py
```

Comment:
Is `non_blocking` usable on all PyTorch/Python versions apart from just PT 1.9 / py38?

Reply:
Made it conditional based on python version.